d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -111,4 +111,4 @@ def on_finish(self, finish_info):
 last_market_stats = rest_market.last_market_stats()
 
 while not aggr.is_finished:
-    sleep(0.5)
+    sleep(0.5)
```

Comment:
As a common practice, it's recommended to always add a newline at the end of the files. That's why some IDEs have the option to automatically add it when saving. Some info I found:
- [Why should text files end with a newline?](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795)
- [What's the point in adding a new line to the end of a file?](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
- [Why is it recommended to have empty line in the end of a source file?](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file)

Reply:
Thanks a lot! This was removed due to some copy and paste. Will add it again.